backend/migrations/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
missing end line

Reply:
I don't write files which are in migrations folder, but they are generated by command "flask db init", and according to the documentation they should be added to repository. So I think i shouldn't change this.

Reply:
So what the problem to add enter? I don't get it

Reply:
Your repository, your project, your code, no1 will forgive you not making something better or cleaner just when someone else started the shitty code.